samples/eShopLite/grafana/dashboards/aspnetcore-endpoint.json

```diff
@@ -0,0 +1,923 @@
+{
```

Comment:
Instead of duplicating these files here, can we just link to them in https://github.com/dotnet/aspire/tree/main/src/Grafana/dashboards?